package.json

```diff
@@ -2,7 +2,7 @@
   "name": "web-admin-dashboard",
   "version": "0.1.0",
   "dependencies": {
-    "@glpi-project/javascript-library-glpi": "^0.0.0",
+    "@glpi-project/javascript-library-glpi": "^0.1.0-beta.5",
```

Comment:
Hi, @hectorerb 
You can pecify a range of prerelease versions.
Use comparisons like > with a prerelease tag.

`"@glpi-project/javascript-library-glpi": ">=0.1.0-beta <=1.0.0"`

Reply:
perfect! I'll do it